pyproject.toml

```
@@ -3,7 +3,7 @@
 
 [tool.black]
 line-length = 120
-target_version = ['py38', 'py39', 'py310']
+target_version = ['3.10', '3.11', '3.12']
```

Shouldn't this use the previous format?
```suggestion
target_version = ['py310', 'py311', 'py312']
```